plugins/newsdownloader.koplugin/main.lua

```diff
@@ -57,6 +57,15 @@ function NewsDownloader:addToMainMenu(menu_items)
                 callback = function() self:setCustomDownloadDirectory() end,
             },
             {
+                text = _("Settings"),
+                callback = function()
+                    UIManager:show(InfoMessage:new{
+                        text = T(_("To change feed (Atom/RSS) sources please manually edit configuration file:\n%1\n\nIt's very simple, contains comments and sample configuration."),
```

Comment:
Having taken a closer look, this is almost exactly the same as the help message underneath. If we were to add this menu item you should stick a variable up above like:

```lua
local help_message = InfoMessage:new{
    text = T(_("News downloader can be configured in the feeds config file:\n%1\n\nIt downloads news items to:\n%2.\n\nTo set you own news sources edit foregoing feeds config file. Items download limit can be set there."),
        feed_config_path,
        news_download_dir_path)
```

There's no reason to have two near-identical messages here.